Code/Framework/AzToolsFramework/AzToolsFramework/Manipulators/AngularManipulatorCircleViewFeedback.cpp

```diff
@@ -89,6 +91,17 @@ namespace AzToolsFramework
                 manipulatorState.m_localPosition + second * view->m_radius * viewScale);
         }
 
+        debugDisplay.SetColor(AZ::Color(1.f));
+
+        const auto drawRadiusSegment = [&](float angle)
```

Comment:
update: Can we please be explicit about what variables we're capturing here instead of the blanked `&` capture (we always try and be explicit about what's captured)

Reply:
Thanks for the quick feedback :) I've just updated the code with your feedback

Reply:
Awesome thanks!